Documents/Automation/EncryptRootVolume/Documents/npark-encryptrootvolume.json

```diff
@@ -171,7 +187,7 @@
       "inputs": {
         "DocumentName": "AWS-DetachEBSVolume",
         "RuntimeParameters": {
-          "VolumeId": "{{extractEBSvolumeID.EBSVolumeID}}"
+          "VolumeId": "{{describeInstanceRootVolume.RootDeviceVolumeId}}"
```

Comment:
You are using different styles for the variables. either do {{variable}} or {{ variable }}.

Reply:
Done.